### PR TITLE
[Fixes #993] Use createLayer configuration to show the create dataset action

### LINKED ISSUE #993: Use createLayer configuration to show the `create dataset` action.
Use the configuration added in https://github.com/GeoNode/geonode-mapstore-client/issues/991 to show the `Create dataset` action here.
ss1

### PR DESCRIPTION
**Pull request by DavidQuartz**

### CODE REVIEW

File: geonode_mapstore_client/client/static/mapstore/configs/localConfig.json [hunk 1]
```diff
@@ -350,6 +350,7 @@
                             "value": "layer",
                             "type": "link",
                             "href": "/createlayer/",
+                            "config": "createLayer",
```
Comment (allyoucanmap) [inline geonode_mapstore_client/client/static/mapstore/configs/localConfig.json:353]: @DavidQuartz could we try to use disableIf instead to add a new param?

`"disableIf": "{((state('settings') && state('settings').createLayer) ? true : false)}",`

I did not test this so please try it locally if it works

**Review by allyoucanmap — CHANGES_REQUESTED** [review]

**Review by allyoucanmap — APPROVED** [review]

**Comment by allyoucanmap** [comment]
please @marthamareal test this on dev when the new bundle is merged and let us know if we can backport on the stable branch 4.x

**Comment by marthamareal** [comment]
@DavidQuartz, Am testing this locally since it's where I can easily update `CREATE_LAYER` setting

I have the setting set to false, however, I can see the `create dataset` link
ss1

**Comment by marthamareal** [comment]
@DavidQuartz, It seems my local environment was not reflecting the changes. After uninstalling and re-installing the client. Tests for this issue passed as expected. It can be safely backported to 4.x.

**Pull request merged**